contributors/design-proposals/seccomp.md

```diff
@@ -112,14 +140,21 @@ specification of seccomp filters, with arguments, operators, and actions.
 Docker allows the specification of a single seccomp filter.  There are
 community requests for:
 
-Issues:
-
 * [docker/22109](https://github.com/docker/docker/issues/22109): composable
   seccomp filters
 * [docker/21105](https://github.com/docker/docker/issues/22105): custom
   seccomp filters for builds
 
-#### rkt / appcontainers
+Implementation details:
```

Comment:
ya I was unsure what the protocol was for when things change but can totally split it out :)